packages/xrpl/test/integration/utils.ts

```diff
@@ -78,29 +231,59 @@ export async function verifySubmittedTransaction(
   }
 }
 
+/**
+ * Sends a test transaction for integration testing.
+ *
+ * @param client - The XRPL client
+ * @param transaction - The transaction object to send.
+ * @param wallet - The wallet to send the transaction from.
+ * @param retry - As of Sep 2022, xrpl.js does not track requests sent in parallel. Our sequence numbers can get off from
+ *               the server's sequence numbers. This is a fix to retry the transaction if it fails due to tefPAST_SEQ.
+ * @param retry.count - How many times the request should be retried.
+ * @param retry.delayMs - How long to wait between retries.
+ * @returns The response of the transaction.
+ */
+// eslint-disable-next-line max-params -- Test function, many params are needed
 export async function testTransaction(
   client: Client,
   transaction: Transaction,
   wallet: Wallet,
-): Promise<void> {
+  retry?: {
+    count: number
+    delayMs: number
+  },
+): Promise<SubmitResponse> {
```

Comment:
I noticed that all of these methods now return the response instead of just void. Are the responses actually used, or is that just so more information is passed around just in case/for debugging purposes?

Reply:
Doesn't look like it's used, so debugging